### PR TITLE
Finding LCM of two numbers using the concept of GCD

### PR DESCRIPTION
**Pull request by anupamahalder**

### CODE REVIEW

File: Mathematics/Find LCM of two numbers.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+class LCM{
+    private:
+        int a, b;
+        //calculate gcd using Euclidean Algorithm
+        int gcd(int a, int b){
+          if(a==0) return b;
+          else if(b==0) return a;
+          if(a==b) return b;
+          if(a>b) gcd(a%b,b);
+          else if(b>a) gcd(a,b%a);
```
Comment (nitishhsinghhh) [inline Mathematics/Find LCM of two numbers.cpp:12]: return gcd(a, b % a);

**Review by nitishhsinghhh — APPROVED** [review]

**Comment by nitishhsinghhh** [comment]
While naming the heading should not include any white space as it would be difficult to link such files in the README.md

**Comment by nitishhsinghhh** [comment]
GCD_two_numbers.cpp ✔
GCD of two numbers.cpp ❌

**Comment by anupamahalder** [comment]
Sure sure i will not add any whitespace in the file name from now.

**Pull request merged**